gulpfile.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const fs = require('fs');
+const gulp = require('gulp');
+const packager = require('electron-packager');
+
+const config = JSON.parse(fs.readFileSync('package.json'));
+const appVersion = config.version;
+const electronVersion = config.devDependencies['electron-prebuilt'].match(/[\d.]+/)[0];
```

Comment:
Just a note that this will not always be accurate if your package.json doesn't specify the latest version of electron-prebuilt that satisifies the semver string. E.g., if the package.json says `"electron-prebuilt": "^0.36.0"`, the Electron version you're using is actually `0.36.12`, so your package.json should be updated to `"electron-prebuilt": "^0.36.12"` for your build to grab the correct Electron version.

Hope that makes sense. 🙃